src/viking.js

```diff
@@ -1,19 +1,93 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor (health,strength){
+    this.health=health
+    this.strength=strength
+  }
+  attack(){
+    return this.strength
+  }
+  receiveDamage(damage){
+    this.health -= damage
+  }
+}
+
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
```

Comment:
Recuerda que el metodo super te da acceso a las propiedades y métodos del padre, por lo que no es necesario repetir las propiedades de health y strenght
```suggestion
class Viking extends Soldier {
  constructor (name) {
    super (health, strength)
    this.name = name
  }
```